From 8177e80a42bfaa5dae8d4b65c5d18b9b8294c087 Mon Sep 17 00:00:00 2001
From: Scott Tsai <scottt.tw@gmail.com>
Date: Sun, 23 Mar 2025 13:16:25 +0800
Subject: [PATCH 04/14] enable hipBLASLt for gfx{1102,1150,1151,1201}

Pytorch upstream should integrate this once the following change
is in a ROCm release:
https://github.com/ROCm/hipBLASLt/pull/1766
---
 aten/src/ATen/Context.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/aten/src/ATen/Context.cpp b/aten/src/ATen/Context.cpp
index 01f223f4e5c..d384b5e6b2f 100644
--- a/aten/src/ATen/Context.cpp
+++ b/aten/src/ATen/Context.cpp
@@ -359,7 +359,7 @@ at::BlasBackend Context::blasPreferredBackend() {
       static const std::vector<std::string> archs = {
           "gfx90a", "gfx942",
 #if ROCM_VERSION >= 60300
-          "gfx1100", "gfx1101", "gfx1200", "gfx1201"
+          "gfx1100", "gfx1101", "gfx1150", "gfx1151", "gfx1200", "gfx1201"
 #endif
 #if ROCM_VERSION >= 60500
           "gfx950"
-- 
2.47.1.windows.2

